.travis.yml

```diff
@@ -70,11 +69,10 @@ jobs:
           curl -L https://github.com/docker/compose/releases/download/1.25.5/docker-compose-$(uname -s)-$(uname -m) > docker-compose
           chmod +x docker-compose
           sudo mv docker-compose /usr/local/bin
-        - yarn do shadowbox/integration_test/run
+        - yarn do shadowbox/docker/build; cd src/shadowbox/integration_test; ./test.sh
```

Comment:
Do we need to use `&&` so we run the test only if building succeeds?

Reply:
Good point. Done